docs/collecting-data/collecting-from-own-applications/java-tracker/previous-versions/java-tracker-v0-12/installation-and-set-up/index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Installation and set-up"
+date: "2022-03-24"
+sidebar_position: 10
+---
+
+## Installing
+
+The Snowplow Java tracker has been built and tested using Java versions 8, 11 and 17, so should work within any Java application built using JDK8 upwards. The Java tracker is also usable from Scala.
+
+The current tracker version is 0.12 ([GitHub](https://github.com/snowplow/snowplow-java-tracker)).
```

Comment:
Should we amend this with `1.0.0` to avoid confusion if someone lands on this page and does not notice it’s inside “previous versions”?

Reply:
I've added `:::info` boxes with links to the current pages. Didn't want to write 1.0.0 in there as it'll be easily overlooked next update